src/components/App/App.tsx

```diff
@@ -0,0 +1,198 @@
+/* eslint-disable jsx-a11y/control-has-associated-label */
+import React, {
+  useState,
+  useContext,
+  useEffect,
+  useRef,
+  useCallback,
+} from 'react';
+
+import { AuthContext } from '../Auth/AuthContext';
+import { Todo } from '../../types/Todo';
+import * as Api from './api/todos';
+import { FilterBy } from '../../types/FilterBy';
+import { TodoList } from '../TodoList';
+import { Footer } from '../Footer';
+
+export const App: React.FC = () => {
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  const user = useContext(AuthContext);
+  const newTodoField = useRef<HTMLInputElement>(null);
+  const [todoList, setTodoList] = useState<Todo[] | null>(null);
+  const [showFooter, setShowFooter] = useState(false);
+  const [filterBy, setFilterBy] = useState<FilterBy>(FilterBy.All);
+  const [error, setError] = useState('');
+
+  const [todoTitle, setTodoTitle] = useState('');
+  const [loadingInput, setLoadingInput] = useState(false);
+  // const [isDeleting, setIsDeleting] = useState(false);
```

Comment:
don't leave unnecessary comments in your code

Reply:
I will delete them!